README.md

```diff
@@ -498,6 +504,12 @@ const tenantsRes = await descopeClient.management.tenant.loadAll();
 tenantsRes.data.forEach((tenant) => {
   // do something
 });
+
+// Search all tenants according to various parameters
+const tenantsRes = await descopeClient.management.tenant.searchAll(['id']);
```

Comment:
not super critical, but maybe change the name to `searchRes` in this line and line 510 - so the entire code block can be copy-pasted and "compile" properly (I think its the first place that repeat variable name, but I looked only in close scope)

Reply:
This is how it is called in tenants  load all, and we return the same object
it is not a search response that has the count with it

Reply:
the problem is that this will now not compile, because you define 2 `const` wit the same name 

until now - a user could copy paste this code and it would work
now - it won't because the repeated const is already declared 

![image](https://github.com/descope/node-sdk/assets/10514677/99d3146d-5c84-4271-b321-436214f025b9)

Reply:
fixed